libraries/network/src/main/kotlin/io/element/android/libraries/network/NetworkModule.kt

```diff
@@ -26,14 +28,15 @@ object NetworkModule {
     @Provides
     @SingleIn(AppScope::class)
     fun providesOkHttpClient(
-        buildMeta: BuildMeta,
         userAgentInterceptor: UserAgentInterceptor,
+        featureFlagService: FeatureFlagService,
     ): OkHttpClient = OkHttpClient.Builder().apply {
         connectTimeout(30, TimeUnit.SECONDS)
         readTimeout(60, TimeUnit.SECONDS)
         writeTimeout(60, TimeUnit.SECONDS)
         addInterceptor(userAgentInterceptor)
-        if (buildMeta.isDebuggable) addInterceptor(providesHttpLoggingInterceptor())
+        val addLogInterceptor = runBlocking { featureFlagService.isFeatureEnabled(FeatureFlags.PrintLogsToLogcat) }
```

Comment:
Is it possible to avoid this runBlocking? I had a quick look, but not sure how to do it simply.